src/component/button.js

```diff
@@ -78,7 +78,13 @@ const glasStyles = StyleSheet.create({
   },
 });
 
-export const GlasButton = ({ onPress, disabled, children, style }) => (
+export const GlasButton = ({
+  onPress,
+  disabled,
+  children,
+  style,
+  textStyle,
```

Comment:
We should avoid setting two style objects in one component. If necessary, I'd prefer pulling the inner components out to make them styleable. But it actually shouldn't be necessary to style the text size for this button.

Reply:
In general the fonts are pretty big. We should try to fit the design into the default viewport size https://github.com/lightninglabs/lightning-app/blob/dd046e284bf13b8c09653eca5c29f5bcb8039d05/public/electron.js#L73-L74

This is what that currently looks like...
![image](https://user-images.githubusercontent.com/1374174/41066755-567da66c-69e3-11e8-8020-ee970742e8bf.png)